docs/Administrator-Guide/Managing-Volumes.md

```diff
@@ -344,25 +280,14 @@ Steps:
            Status of volume: r2
            Gluster process                        Port    Online    Pid
            ------------------------------------------------------------------------------
-           Brick Server1:/home/gfs/r2_5            49156    Y    5731 <<<---- new brick is online
+           Brick Server1:/home/gfs/r2_5            49156    Y    5731 <---- new brick is online
            Brick Server2:/home/gfs/r2_1            49153    Y    5354
            Brick Server1:/home/gfs/r2_2            49154    Y    5365
            Brick Server2:/home/gfs/r2_3            49155    Y    5376
 
-    - Users can track the progress of self-heal using: `gluster volume heal [volname] info`.
-      Once self-heal completes the changelogs will be removed.
-
-            # getfattr -d -m. -e hex /home/gfs/r2_1
-            getfattr: Removing leading '/' from absolute path names
-            # file: home/gfs/r2_1
-            security.selinux=0x756e636f6e66696e65645f753a6f626a6563745f723a66696c655f743a733000
-            trusted.afr.r2-client-0=0x000000000000000000000000 <<---- Pending changelogs are cleared.
-            trusted.afr.r2-client-1=0x000000000000000000000000
-            trusted.gfid=0x00000000000000000000000000000001
-            trusted.glusterfs.dht=0x0000000100000000000000007ffffffe
-            trusted.glusterfs.volume-id=0xde822e25ebd049ea83bfaa3c4be2b440
+    - Users can track the progress of self-heal using: `gluster volume heal [volname] info`, or by checking the size of the new brick.
 
-    - `# gluster volume heal <VOLNAME> info` will show that no heal is required.
+    - `# gluster volume heal <volname> info` will show that no heal is required when the data on the replaced brick has been migrated completely.
```

Comment:
```suggestion
    - `# gluster volume heal <volname> info` will show that no heal is required when the data is fully synced to the replaced brick.
```

The data will be synced from the other good brick(s) of the replica to the replaced one. The current statement gives the impression that the data will be migrated from the old brick to the newly replaced brick.